test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -448,6 +448,35 @@
               "No exception thrown even though no file existed for copying")
           (fs/copy copy cacert))))))
 
+(deftest retrieve-ca-crl!-test
+  (testing "CRL file copied when it doesn't already exist"
+    (let [tmp-confdir (fs/copy-dir confdir (ks/temp-dir))
```

Comment:
My understanding is that a single temporary directory will be created and used for the following examples.  Will this mean the results will be indeterminate if the examples are run concurrently?

I need to learn more about how tests execute in Clojure and if it's common to run them concurrently or if we plan to.